server/utils.py

```diff
@@ -18,8 +21,68 @@ class Settings(BaseSettings):
 
 
 def get_input_topic(job_id: str):
-    return f"adala-input-{job_id}"
+    topic_name = f"adala-input-{job_id}"
+
+    # same logic as get_output_topic
+
+    settings = Settings()
```

Comment:
Can we separate out the logic for topic creation into its own function to not have a duplicate? get_input / output can just call that function

Reply:
yep